tests/qm-connectivity/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -eux
+
+source ../e2e/lib/utils
+source ../e2e/lib/tests
+
+test_hirte_list_all_units
```

Comment:
Since these test scripts are mainly all bash, consider adding a github-action (or other harness) job to run shellcheck over them.  Since most of these scripts are new-ish, now is an ideal time to add such checks.

Reply:
Correct

Reply:
Ack